src/github.com/getlantern/flashlight/config/bootstrap.go

```diff
@@ -102,14 +103,15 @@ func MakeInitialConfig(configPath string) error {
 	bytes, err := fs.GetIgnoreLocalEmpty("lantern.yaml")
```

Comment:
Are we ever reading the local filesystem for this? Why? Shouldn't the embedded be the default?